### PR TITLE
[IMP] l10n_in*: cleanup invisible fields

### PR DESCRIPTION
**Pull request by hamo-odoo**
see https://github.com/odoo/enterprise/pull/69198

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by smetl** [comment]
@hamo-odoo You need to retarget to master because you are breaking the whole stable policy here

File: addons/l10n_in_ewaybill_stock/views/l10n_in_ewaybill_views.xml [hunk 1]
```diff
@@ -129,7 +120,6 @@
                             <field name="move_ids" mode="tree,kanban" force_save="1" readonly="state != 'pending'">
                                 <tree editable="bottom" create="0" delete="0">
                                     <field name="company_currency_id" column_invisible="1"/>
```
Comment (hamo-odoo) [inline addons/l10n_in_ewaybill_stock/views/l10n_in_ewaybill_views.xml:122]: @smetl Do you know a way to get rid of this `company_currency_id `? The issue on removing it is that we currency symbol is no longer visible on the tree view

Comment (jco-odoo) [reply addons/l10n_in_ewaybill_stock/views/l10n_in_ewaybill_views.xml:122]: @hamo-odoo I think we need it here anyways as it is the currency_field of ewaybill_priceunit

File: addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml [hunk 2]
```diff
@@ -20,11 +20,6 @@
                     <field name="warning_message"/>
                 </div>
                 <sheet>
-                    <field name="related_move_id" invisible="1"/>
```
Comment (jco-odoo) [inline addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml:23]: @hamo-odoo It would be good to check that the wizard is still working with this.

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**